src/libraries/System.Private.CoreLib/src/System/Runtime/JitInfo.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime
+{
+    public static partial class JitInfo
+    {
+        private const long TicksPerMillisecond = 10000;
+        private const long TicksPerSecond = TicksPerMillisecond * 1000;
+
+        // "Frequency" stores the frequency of the high-resolution performance counter,
+        // if one exists. Otherwise it will store TicksPerSecond.
+        // The frequency cannot change while the system is running,
+        // so we only need to initialize it once.
+        public static readonly long Frequency = System.Diagnostics.Stopwatch.QueryPerformanceFrequency();
```

Comment:
Why can't you just use Stopwatch.Frequency?

Reply:
That's a misread on my part. I read `Stopwatch.Frequency` as `private` instead of `public`. I'll change it.